projects/centre_console/src/power_fsm.c

```diff
@@ -64,6 +65,8 @@ static void prv_state_fault_output(Fsm *fsm, const Event *e, void *context) {
   } else {
     power_fsm->previous_state = POWER_STATE_OFF;
     event_raise(CENTRE_CONSOLE_POWER_EVENT_OFF, 0);
```

Comment:
We don't need to do this anymore eh? Since we're locking up centre console and letting power distro turn off whatever needs turned off.

Reply:
I remember why I kept them now. It was to passed a test. Should I just get rid of part of the test?

Reply:
Yeah just get rid of that part